.gitignore

```diff
@@ -16,7 +16,6 @@ yarn-error.log*
 
 # Editor directories and files
 .idea
-.vscode
```

Comment:
I suggest to exclude all vs code specific files and only add the ones we really want

```
.vscode/*
!.vscode/settings.json
!.vscode/extensions.json
```

Reply:
It's intentional to share these settings.